internal/command/binding_create.go

```diff
@@ -102,6 +107,8 @@ func newBindingCreateCommand(p *KameletPluginParams) *cobra.Command {
 	flags.StringVar(&service, "service", "", "Uses a Knative service as binding sink.")
 	flags.BoolVar(&force, "force", false, "Apply the changes even if the binding already exists.")
 	flags.StringArrayVar(&properties, "property", nil, `Add a source property in the form of "<key>=<value>"`)
+	flags.StringVar(&cloudEventsSpecVersion, "cloudEventsSpecVersion", "", "Customize cloud events spec version provided to the binding sink.")
+	flags.StringVar(&cloudEventsType, "cloudEventsType", "", "Customize cloud events type provided to the binding sink.")
```

Comment:
The `kn` has utilized `--ce-override` flag for customization purposes. I wonder if that wouldn't be helpful in this case as well. 

At least I'd like to propose shortening the name to e.g. `ce-version` or `ce-spec` and `ce-type`. Wdyt?

```
 --ce-override stringArray   Cloud Event overrides to apply before sending event to sink. Example: '--ce-override key=value' You may be provide this flag multiple times. To unset, append "-" to the key (e.g. --ce-override key-).
```

Reply:
yes, good point. `ce-spec` and `ce-type` are much more user friendly

Reply:
I'll also have a look at `ce-override` option